src/Nri/Ui/SegmentedControl/V10.elm

```diff
@@ -0,0 +1,305 @@
+module Nri.Ui.SegmentedControl.V10 exposing
+    ( Option, view
+    , SelectOption, viewSelect
+    , Width(..)
+    )
+
+{-| Changes from V9:
+
+  - hides non-displayed content rather than fully removing from the DOM, allowing for the segmented control to be used as a sticky-header that doesn't lose scroll position
+  - :skull: removes NavConfig and SelectConfig
```

Comment:
*definitely* a nitpick, but if you want this  to show up as 💀 it needs to be the literal emoji, as the package site doesn't parse Slack-style emoji names.